tests/test_study.py

```diff
@@ -407,7 +404,7 @@ def func_none(_):
         frozen_trial = study._storage.get_trial(trial._trial_id)
 
         expected_message = (
-            "Setting status of trial#3 as TrialState.FAIL because the returned "
+            "Trial 3 failed, because the returned "
             "value from the objective function cannot be casted to float. "
```

Comment:
```suggestion
            "value from the objective function cannot be cast to float. "
```

Reply:
Sorry for the confusion. My concern is already addressed in the commit just submitted.